scripts/qa_crowbarsetup.sh

```diff
@@ -2368,7 +2368,9 @@ function enable_ssl_generic
         ;;
     esac
     $p "$a['ssl']['generate_certs']" true
-    $p "$a['ssl']['insecure']" true
+    if [ "$service" != "rabbitmq" ] ; then
```

Comment:
we could have a validation in crowbar that makes sure that the value is always 'true' and produce an error with a link to the bug otherwise.